frontend/packages/ux-editor/src/components/config/editModal/EditDataModelBinding/EditBinding/SelectDataModelBinding.tsx

```diff
@@ -42,13 +53,18 @@ export const SelectDataModelBinding = ({
           className={classes.selectDataModel}
           {...fieldProps}
           label={t('ux_editor.modal_properties_data_model_binding')}
-          id={selectedDataModel}
+          id={id}
           onChange={(e) => fieldProps.onChange(e.target.value)}
           size='small'
         >
-          {dataModels.map((element) => (
-            <option key={element} value={element}>
-              {element}
+          {defaultDataTypeName && (
+            <option key={defaultDataTypeName} value={defaultDataTypeName}>
+              {defaultDataTypeName}
+            </option>
+          )}
+          {bindableDataTypes.map((dataType) => (
+            <option key={dataType.id} value={dataType.id}>
+              {dataType.id}
             </option>
           ))}
```

Comment:
What do you think if we show a message when there is no datamodell to show? like this maybe: 😄 
 ```
   <option value='' disabled selected hidden></option>
          {bindableDataTypes.length > 0 ? (
            bindableDataTypes.map((dataType) => (
              <option key={dataType.id} value={dataType.id}>
                {dataType.id}
              </option>
            ))
          ) : (
            <option key='noDataTypes' value='' disabled>
              {'Ingen datamodell tilgjengelig'}
            </option>
          )}
```

Reply:
I agree but I think UX should be involved in this, and since this is how it works currently on prod I think we should maybe make this part of another issue? Do you agree?

Reply:
Ok, then maybe this can be fixes in another issue when we get ok from UX 🙂